CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE code repo](README.md), check out the [Contributor's Guide](https://github.com/o3de/community/blob/main/CONTRIBUTING.md).
+
+The [O3DE community repo](https://github.com/o3de/community) contains information about how to get started, how the community organizes, and more.
+
+To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/blob/development/CODE_OF_CONDUCT.md) first. 
+
+## Making contributions with the Developer Certificate of Origin (DCO)
+
+When contributing, your pull requests will require that you have agreed to our DCO found here: [Devloper Certificate of Origin](https://developercertificate.org/)
```

Comment:
minor: IMHO This need a little more context about what this is? It just links to DCO but for folks new to open-source/DCO it probably just needs a little expansion ie 

"...agree to our Developer Certification of Origin(https://developercertificate.org/) statement. All commits require the [--signoff](https://git-scm.com/docs/git-commit) flag to show DCO compliance.

Reply:
Added, good call out.